src/site/content/en/metrics/inp/index.md

```diff
@@ -158,56 +157,22 @@ INP can be measured both in [the field](/lab-and-field-data-differences/#field-d
-  - CrUX API via `interaction_to_next_paint`.
-  - CrUX Dashboard.
-- [`web-vitals` JavaScript library](https://github.com/GoogleChrome/web-vitals).
-
-### Lab tools
+### In the field
 
-- Lighthouse Panel in DevTools, available in "Timespan Mode".
-- [Lighthouse npm module](https://www.npmjs.com/package/lighthouse).
-- [Lighthouse User Flows](/lighthouse-user-flows/).
-- [Web Vitals extension for Chrome](https://chrome.google.com/webstore/detail/web-vitals/ahfhijdlegdabablpippeagghigmibma).
+Ideally, your journey in optimizing INP will start with field data. At its best, field data from a Real User Monitoring (RUM) provider will give you not only a page's INP value, but also contextual data that highlights what specific interaction was responsible for the INP value itself, whether the interaction occurred during or after page load, the type of interaction (click, keypress, or tap), and other valuable information.
 
-### Measure interactions in JavaScript
-
-INP is calculated based on interaction events. The following example shows how to create a [`PerformanceObserver`](https://developer.mozilla.org/docs/Web/API/PerformanceObserver) to log interaction entries to the console:
-
-```js
-new PerformanceObserver((entryList) => {
-  for (const entry of entryList.getEntries()) {
-    if (entry.interactionId) {
-      const duration = entry.processingEnd - entry.startTime;
-      console.log('Interaction:', entry.name, duration, entry);
-    }
-  }
-}).observe({type: 'event', buffered: true, durationThreshold: 16});
-```
-
-### Measure INP in JavaScript
-
-To measure INP in JavaScript, it's advised that you use the [`web-vitals` JavaScript library](https://github.com/GoogleChrome/web-vitals), which exports an `onINP` function to handle the aggregation of each individual interaction. You can then get a page's INP by passing a function to the `onINP` method:
-
-```js
-import {onINP} from 'web-vitals';
+{% Aside 'objective' %}
+**Read to learn more:**&nbsp;[Find slow interactions in the field](/find-slow-interactions-in-the-field/).
+{% endAside %}
 
-onINP(({value}) => {
-  // Log the value to the console, or send it to your analytics provider.
-  console.log(value);
-});
-```
+If you're not relying on a RUM provider to get field data, the [INP field data guide](/find-slow-interactions-in-the-field/) advises [using the Chrome User Experience Report (CrUX) via PageSpeed Insights](/find-slow-interactions-in-the-field/#get-field-data-quickly-with-crux) to help fill in the gaps. CrUX is the official dataset of the Core Web Vitals program and provides a high-level summary of metrics for millions of websites, including INP. However, CrUX often does not provide the contextual data you'd get from a RUM provider to help you to analyze issues. Because of this, we still recommend that sites use a RUM provider when possible, or implement their own RUM solution to supplement what is available in CrUX.
```

Comment:
I'm not loving this framing.

I think we should say:
- CrUX is best for the score. To me, as the official dataset of the CWV initiative that should be the starting point. It's the most accurate number IMHO given the limitations on Web APIs used by RUM and given that NONE of the RUM providers measure INP throughout the full page lifecycle (many stop after first interaction).
- RUM allows you to get more detailed info than CrUX can provide - including attribution details (e.g. element clicked for the back INP).
- Lab should then be used to replicate (ideally with RUM as a clue as to which pages, and while elements, are causing the slowness).